test/e2e/specs/liquidation.spec.js

```diff
@@ -477,7 +477,7 @@ describe('Wallet App Test Cases', () => {
       cy.skipWhen(AGORIC_NET === networks.LOCAL);
 
       const propertyName = 'book0.collateralAvailable';
-      const expectedValue = '9.659302 ATOM';
+      const expectedValue = '9.659301 ATOM';
```

Comment:
can you explain why you had to update this value?

Reply:
When matching auction values, the expected value for `collateralAvailable` is `9.659301 ATOM`. It is rarely `9.659302 ATOM`

Reply:
if it can have two close possible values, i think a better approach would be to check that the difference is within a certain limit:
```
expect(Math.abs(expectedValue - foundValue)).toBeLessThan(0.0001);
```

Reply:
It is a generic function that compares string values. I don't want to mess it up because it is being used in multiple places. 
`expect(Math.abs(expectedValue - foundValue)).toBeLessThan(0.0001);` incorrect as I said we compare strings. Strings have token denominations as well.